### DISCUSSION TITLE
Why is an Excel Hyperlink created?

### DISCUSSION THREAD

**Discussion by abt-effadj** [dfinke/ImportExcel#1661] [Q&A] [answered]
I have text data values (see below) and when processed by Export-Excel, most become Excel hyperlinks which I do not want.  How can I prevent this from happening?  I tried a few things, but I could not change that behavior.  Thank you (and for your great ImportExcel module!)

Asia Pacific:Hong Kong
Asia Pacific:New South Wales
Asia Pacific:Pune
Asia Pacific:Seoul
Asia Pacific:Singapore
Asia Pacific:Tokyo, Saitama
Canada:Toronto
Europe:Cardiff
Europe:Frankfurt
Europe:Ireland
Europe:London
Europe:Netherlands
Europe:Norway
Europe:Paris
South America:Sao Paulo State
US:California
US:Illinois
US:Iowa
US:Texas
US:Virginia
US:Washington

**Comment by dfinke** [comment, marked as answer]
try `-NoHyperLinkConversion *` or name the properties individually

**Comment by abt-effadj** [comment]
Doug…Worked great!  I am glad I asked you.

Have a great weekend.  Again, thanks for the module!

Regards,

David
________________________________
David J Effa
Sr. Specialist Systems Support
Business Technology Services
Global Windows Implementation
Abbott<http://www.abbott.com/>
Bldg. AP14B
100 Abbott Park Rd.
Abbott Park, IL 60064
Work +1 224-668-4382
Cell   +1 224-448-6334
***@***.******@***.***>
https://www.linkedin.com/in/davideffa/
[logo image]
________________________________
This communication may contain information that is proprietary, confidential, or exempt from disclosure. If you are not the intended
recipient, please note that any other dissemination, distribution, use or copying of this communication is strictly prohibited. Anyone
who receives this message in error should notify the sender immediately by telephone or by return e-mail and delete it from their
computer.
